swimlane/core/fields/attachment.py

```diff
@@ -55,6 +55,10 @@ def get_initial_elements(self):
 
         return [self.cast_to_python(raw) for raw in raw_value]
 
+    def get_batch_representation(self):
```

Comment:
were you able to get attachments working with this?

because the PATCH endpoint for records goes through batch modification I'm getting a `ModificationError:5014 (Cannot modify attach field. Invalid type - Attachment.)` upon attempting

Reply:
I got that same error.  I looked at what was sent and it seemed correct.  I opened this as a result: https://swimlane.atlassian.net/browse/SPT-6435

Reply:
👍 looks like both attachments and comments will throw there as we don't allow batch for either currently on the backend

I believe the preferred comments route for now is to go through the comments specific endpoint `POST /app/{appId}/record/{recordId}/{fieldId}/comment`

Reply:
With comments, I didn't get an error it just didn't work. I opened https://swimlane.atlassian.net/browse/SPT-6436 for that one. I just made these changes assuming everything could go through that patch endpoint.  If that ends up not being the case feel free to change again or point me in the right direction and I'm happy to help.